Changelog.md

```diff
@@ -7,7 +7,6 @@ Compiler Features:
  * Build System: LLL is not built anymore by default. Must configure it with CMake as `-DLLL=ON`.
  * Code generator: Do not perform redundant double cleanup on unsigned integers when loading from calldata.
 
-
```

Comment:
Undo this line change in `Changelog.md`, as it's not having any meaning. :-)

Reply:
Fixed.